.gitlab-ci.yml

```
@@ -4,12 +4,18 @@ stages:
   - mirror
 
 mirror-from-github:
+    retry: 2
     stage: mirror
     rules:
           - if: $TOKEN
     before_script:
         - source /etc/os-release && zypper ar http://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_${VERSION}/SUSE:CA.repo
-        - zypper -n in git ca-certificates-suse
+        - |
+          for i in {1..3}; do
+              zypper -n in git ca-certificates-suse
+              zypper ref
+              sleep 1
```

It is bad to sleep 1 second *unconditionally*, even when everything works fine. This loop will also *always* make 3 attempts, even if none of the commands in it fail. Supposedly just a break in case the commands succeeded is missing:

```suggestion
              zypper -n in git ca-certificates-suse && zypper ref && break
              sleep 1
```

Or should `zypper ref` only be done when the first command failed? Like this:

```suggestion
              zypper -n in git ca-certificates-suse && break
              zypper ref
              sleep 1
```